app/searches/lookup_drugs.rb

```diff
@@ -29,8 +30,9 @@ def self.match_search_terms_to_objects(search_terms, scope)
     search_terms = search_terms.dup
     drug_results = DataModel::Drug.send(scope).where(name: search_terms)
     search_terms = search_terms - drug_results.map(&:name)
-    drug_alias_results = DataModel::DrugClaimAlias.send(scope).where(alias: search_terms)
+    drug_alias_results = DataModel::DrugAlias.send(scope).where(alias: search_terms)
     search_terms = search_terms - drug_alias_results.map(&:alias)
+    #Can we get rid of this case
```

Comment:
It is my opinion that we can drop this, but I left the '?' because I'd like the others to weigh in on this first. I can't think of a reason we would want to keep this, but @acoffman wrote most of the search engine, and then I used that as a template for adding drug search.